frontend/src/services/searchfetcher/SearchFetcher.ts

```diff
@@ -0,0 +1,22 @@
+// TODO: change to the actual API response
+export interface Opportunity {
```

Comment:
Would it make sense to create a ./models directory to store objects like Opportunity that may be re-used in other parts of the codebase?

Reply:
Sammy started a `types` directory, while I agree the `models` might be a little better long term if we have a bunch of interfaces/types that correspond to API data structures, for now I'm just going to roll with the `types` folder